lib/cmd/fh3/admin/domains/check.js

```diff
@@ -23,8 +23,10 @@ module.exports = {
     return "/box/api/domains/check?domain=" + domain;
   },
   'method' : 'get',
-  'postCmd': function(params, cb) {
-    params._table = common.createObjectTable(params);
+  'postCmd': function(argv, params, cb) {
+    if (!argv.json) {
+      params._table = common.createObjectTable(params);
+    }
     return cb(null, params);
```

Comment:
I understand your concern. Following few points. 

* So, are not all commands that we have a table as result, however, if we change the cb as you suggested all commands need be changed to follow it;
* Since it is not for all, is really a good idea no make this explicit for each command in the code? Is not better for now we keep this as a specification of the each command?

What do you think about we follow with this way that we solved here( simple way ) and in the future when we finish this hard part of the technical debts and we have more visibility we can come back to check the best approach to implement it as a global implementation as you are suggesting?